src/handlers/positions/handler.js

```diff
@@ -221,9 +231,11 @@ const positions = async (error, messages) => {
       if (!Kafka.Consumer.isConsumerAutoCommitEnabled(kafkaTopic)) {
         await consumer.commitMessageSync(message)
       }
+      histTimerEnd({success: true, fspId: Config.INSTRUMENTATION_METRICS_LABELS.fspId})
```

Comment:
Should be removed since this is already handled on line 238.

Reply:
fixed